index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
html lang 속성은 ko로 적용되는게 좋아요~
```suggestion
<html lang="ko">
```